Patch status: pending being looked at upstream

Issue[1][2][3] also exist in chromium itself[4], and unclear
why this started happening only with ninja-1.12.0 at the moment.

Just a quickfix, likely not correct and seems there may be
further unresolved race issues[5]. Arguably better to not
use ninja-1.12.0 for this yet (1.11.1 or samurai are fine).

[1] https://bugs.gentoo.org/930107
[2] https://bugreports.qt.io/browse/QTBUG-124375
[3] https://github.com/ninja-build/ninja/issues/2417
[4] https://bugs.gentoo.org/930112
[5] https://bugs.gentoo.org/930107#c17
--- a/src/3rdparty/chromium/content/browser/BUILD.gn
+++ b/src/3rdparty/chromium/content/browser/BUILD.gn
@@ -198,2 +198,3 @@
     "//components/services/storage/public/mojom",
+    "//components/spellcheck:buildflags",
     "//components/sqlite_proto",
--- a/src/3rdparty/chromium/extensions/browser/api/declarative_net_request/BUILD.gn
+++ b/src/3rdparty/chromium/extensions/browser/api/declarative_net_request/BUILD.gn
@@ -22,2 +22,3 @@
     "//content/public/browser",
+    "//components/web_cache/public/mojom",
     "//extensions/common",
